samples/StaticHtmlComposites/package.json

```diff
@@ -25,7 +25,7 @@
     "@azure/communication-react": "1.7.0-beta.1",
     "@azure/communication-common": "3.0.0-beta.1 || ^2.2.1",
     "@azure/communication-calling": "1.15.1-beta.1 || >=1.14.1",
-    "@azure/communication-chat": "1.3.2-beta.3 || >=1.2.0",
+    "@azure/communication-chat": "1.3.2-beta.3 || >=1.3.1",
```

Comment:
It looks like 1.2.0 is required for stable version. Was this changed?
https://www.npmjs.com/package/@azure/communication-react?activeTab=code
![image](https://github.com/Azure/communication-ui-library/assets/98852890/e1be146b-a6f9-4a1c-9501-278cf3570d9c)

Reply:
The rush install is using 1.3.1 for the common versions
The range here just changes this to ensure it's at least 1.3.1 in the package.